### PR TITLE
fix: load after `nvim-web-devicons` (fixes #31)

### LINKED ISSUE #31: Conflict with `nvchad_ui.icons` (`nvim-web-devicons`)
Loading this plugin before `nvim-web-devicons` will result in a ugly error

image

but just adding

```lua
after = "nvim-web-devicons",
```

to the plugin loading, fixes the error. Might open a PR to modify the `README.md`.

### PR DESCRIPTION
**Pull request by sRavioli**
fixes the conflict with `nvim-web-devicons` by loading `barbecue.nvim` after it.

### CODE REVIEW

**Comment by utilyre** [comment]
Doesn't the `requires = "kyazdani42/nvim-web-devicons"` part do the same thing?

**Comment by sRavioli** [comment]
kinda, `requires = "kyazdani42/nvim-web-devicons"` tells packer that `barbecue.nvim` needs that plugin to work, but not the specific order in which to load the plugins. We do that with the `after` command, this way we avoid any conflict with `nvim-web-devicons`.

I saw it discussed in the NvChad repo ([here](https://github.com/NvChad/NvChad/issues/1362#issuecomment-1213641536))

File: README.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ use {
     "smiteshp/nvim-navic",
     "kyazdani42/nvim-web-devicons", -- optional
   },
+  after = "nvim-web-devicons",
```
Comment (utilyre) [inline README.md:37]:
```suggestion
  after = "nvim-web-devicons", -- NOTICE: keep this if you're using NvChad
```

**Comment by utilyre** [comment]
Please rebase towards `dev` and force push.

**Comment by sRavioli** [comment]
> Please rebase towards `dev` and force push.

i changed the PR target, i wrote the PR from GitHub since it was a oneliner.

**Review by utilyre — CHANGES_REQUESTED** [review]

**Comment by utilyre** [comment]
> > Please rebase towards `dev` and force push.
> 
> i changed the PR target, i wrote the PR from GitHub since it was a oneliner.

I don't know how to do this on the github web, but please clone your fork (with `--branch patch-1`) and run the following

```bash
git rebase remotes/origin/dev
git push -f
```

**Review by utilyre — APPROVED** [review]

**Comment by sRavioli** [comment]
ok soo, should i still clone and stuff?

**Comment by utilyre** [comment]
> ok soo, should i still clone and stuff?

Yep, because your pr contains duplicated commits from `dev` which hasn't been merged yet.

**Comment by sRavioli** [comment]
okok, i just did

**Comment by utilyre** [comment]
Thanks!

**Pull request merged**